src/libraries/Common/src/System/IO/PathInternal.cs

```diff
@@ -50,30 +50,26 @@ internal static int GetCommonPathLength(string first, string second, bool ignore
         /// <summary>
         /// Gets the count of common characters from the left optionally ignoring case
         /// </summary>
-        internal static unsafe int EqualStartingCharacterCount(string? first, string? second, bool ignoreCase)
+        internal static int EqualStartingCharacterCount(string? first, string? second, bool ignoreCase)
```

Comment:
```suggestion
        private static int EqualStartingCharacterCount(string? first, string? second, bool ignoreCase)
```

Reply:
thanks! addressed in the latest commit

Reply:
Ahh didn't see there's a reference in `PathInternal_Windows_Tests`. 🤦‍♂️